en/docs/getting-started/overview.md

```diff
@@ -1,99 +1,113 @@
 
-WSO2 API Manager provides a state-of-the-art web interface called **WSO2 API Publisher** for API development and management. It is a structured GUI designed for API creators to develop, document, scale and version APIs, while also facilitating more API management-related tasks such as publishing API, monetizating APIs, and promoting.
+WSO2 API Manager provides a state-of-the-art web interface called **WSO2 API Publisher** for API development and management. It is a structured GUI designed for API creators to develop, document, scale and version APIs, while also facilitating more API management-related tasks such as publishing API, monetizing APIs, and promoting.
 
-The diagram above shows the common lifecycle activities of an API developer/manager.
+The diagram above shows the common lifecycle activities of an API developer/manager.
 
-#### Developer Portal
+#### API Developer Portal
 
-![]({{base_path}}/assets/attachments/103327648/126556772.png)
-
-The Developer Portal is a state-of-the-art web interface that allows API publishers to host and advertise their APIs while allowing API consumers to [self register]({{base_path}}/learn/consume-api/customizations/customizing-the-developer-portal/enabling-or-disabling-self-signup), discover, evaluate, subscribe to and consume APIs.
+![API Developer Portal Component Architecture]({{base_path}}/assets/attachments/103327648/126556772.png)
 
+The Developer Portal is a state-of-the-art web interface that allows API publishers to host and advertise their APIs while allowing API consumers to [self-register]({{base_path}}/develop/customizations/customizing-the-developer-portal/enabling-or-disabling-self-signup), discover, evaluate, subscribe to and consume APIs.
 
 The diagram above shows common API consumer lifecycle activities:
 
 #### API Gateway
 
-The API Gateway is a runtime, backend component (an [API proxy](https://docs.wso2.com/display/EI611/Working+with+APIs) ) developed using [WSO2 ESB]{{base_path}}/getting-started/about-this-release/). API Gateway secures, protects, manages, and scales API calls. It intercepts API requests, applies policies such as throttling and security using handlers, and manages API statistics. Upon validation of a policy, the Gateway passes web service calls to the actual backend. If the service call is a token request, the Gateway passes it directly to the [Key Manager](#key-manager).
+The API Gateway is a runtime, backend component (an [API Proxy](https://docs.wso2.com/display/EI611/Working+with+APIs)) developed using [WSO2 ESB]({{base_path}}/getting-started/about-this-release/). API Gateway secures, protects, manages, and scales API calls. It intercepts API requests, applies policies such as throttling and security using handlers, and manages API statistics. Upon validation of a policy, the Gateway passes web service calls to the actual backend. If the service call is a token request, the Gateway passes it directly to the [Key Manager](#key-manager).
```

Comment:
In this sentence, we have referenced the `API Proxy` with the EI documentation. Furthermore, we have pointed the reference of the `WSO2 ESB` to the `About this Release` page. Appreciate if you can share the correct paths to be referred

Reply:
This seems to be more relevant https://docs.wso2.com/display/EI660/Working+with+Proxy+Services+. Also please use the term WSO2 EI instead of WSO2 ESB.